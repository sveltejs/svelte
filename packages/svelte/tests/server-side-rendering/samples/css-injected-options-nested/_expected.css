
.foo.svelte-1nvcr6w {
	color: red;
}
