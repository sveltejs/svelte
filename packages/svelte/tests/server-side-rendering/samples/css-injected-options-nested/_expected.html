<!--[--><div class="bar svelte-1nvcr6w">bar</div><!----> <div class="foo svelte-sg04hs">foo</div><!--]-->
